### DISCUSSION TITLE
Yet another GUI, based on Qt

### DISCUSSION THREAD

**Discussion by ghost** [acheong08/EdgeGPT#365] [General]
https://github.com/InterestingDarkness/EdgeGPT
Discussions (in Chinese) https://github.com/viopsa233/aitesters-forum-backup/discussions/25

**Comment by Nu11Const** [comment]
Why free sydney?x2

**Comment by ghost** [comment]
No longer maintained, a maintained version https://github.com/juzeon/SydneyQt
